app/views/tracks/exercises/show/_iteration_section.html.haml

```diff
@@ -0,0 +1,120 @@
+          = link_to "#" do
+            this article
+            = icon "external-link", "Opens in a new tab"
+      .status.failed Tests Failed
+
+    - elsif iteration.status.to_sym == :essential_automated_feedback
+      .info
+        %h2 Your solution worked, but you can take it further…
+        %p
+          = succeed(".") do
+            We’ve analysed your solution and have
+            %span.essential= pluralize iteration.num_essential_automated_comments, "essential improvements"
+            - if iteration.has_actionable_automated_feedback?
+              - if iteration.has_non_actionable_automated_feedback?
+                ,
+              - else
+                and
+              %span.actionable= pluralize iteration.num_actionable_automated_comments, "recommendation"
+            - if iteration.has_non_actionable_automated_feedback?
+              and
+              %span.non-actionable= pluralize iteration.num_non_actionable_automated_comments, "additional comment"
+
+          Address the essential improvements before proceeding.
+
+      .status.passed Tests Passed
+
+    - elsif iteration.status.to_sym == :actionable_automated_feedback
+      - if exercise.concept_exercise?
+        .info
+          %h2 Your solution is good enough to continue!
+          %p
+            = succeed(".") do
+              We’ve analysed your solution and have
+              %span.actionable= pluralize iteration.num_actionable_automated_comments, "recommendation"
+              - if iteration.has_non_actionable_automated_feedback?
+                and
+                %span.non-actionable= pluralize iteration.num_non_actionable_automated_comments, "additional comment"
+
+            You can either continue or address the recommendations first - your choice!
+        .status.passed Tests Passed
+      - else
+        .info
+          %h2 Your solution worked, but you can take it further…
+          %p
+            = succeed(".") do
+              We’ve analysed your solution and have
+              %span.actionable= pluralize iteration.num_actionable_automated_comments, "recommendation"
+              - if iteration.has_non_actionable_automated_feedback?
+                and
+                %span.non-actionable= pluralize iteration.num_non_actionable_automated_comments, "additional comment"
+
+            We suggest addressing the recommendations before proceeding.
+        .status.passed Tests Passed
+
+    - elsif iteration.status.to_sym == :non_actionable_automated_feedback
+      .info
+        %h2 Your solution looks great!
+        %p
+          We’ve analysed your solution and not found anything that needs changing.
+          We do have
+          %span.non-actionable= pluralize iteration.num_non_actionable_automated_comments, "additional comment"
+          that you might like to check.
+          - if exercise.practice_exercise?
+            Consider working with a mentor to make it even better.
```

Comment:
Could this contain a link to request mentoring, or do you feel like that is better left in a separate block below.

Reply:
Probably better to be below still, as there isn't a mentoring "page" any more, just the multipart button. It will also have this above it, so the mentoring will be called out.

<img width="536" alt="Screenshot 2021-02-23 at 12 52 12" src="https://user-images.githubusercontent.com/286476/108845975-f8f19d80-75d5-11eb-9a28-59e6a241db40.png">

Reply:
(This is what goes into the unimplemented "nudge" template)

Reply:
👍